packages/tss-client/package.json

```diff
@@ -20,7 +20,7 @@
   "author": "Web3Auth",
   "license": "ISC",
   "dependencies": {
-    "@toruslabs/eccrypto": "^4.0.0",
+    "@toruslabs/eccrypto": "^5.0.4",
     "@types/chrome": "^0.0.268",
```

Comment:
this types package should be a dev dependency

Reply:
It has been moved.